packages/gateway/src/features/user/root-resolvers.ts

```diff
@@ -30,23 +30,20 @@ import {
 } from '@gateway/graphql/schema'
 import { logger, isBase64FileString } from '@opencrvs/commons'
 import { checkVerificationCode } from '@gateway/routes/verifyCode/handler'
-import { UserInputError } from 'apollo-server-hapi'
+
 import fetch from '@gateway/fetch'
 import { validateAttachments } from '@gateway/utils/validators'
 import { postMetrics } from '@gateway/features/metrics/service'
 import { uploadBase64ToMinio } from '@gateway/features/documents/service'
 import { rateLimitedResolver } from '@gateway/rate-limit'
+import { UserInputError } from '@gateway/utils/graphql-errors'
 
 export const resolvers: GQLResolver = {
   Query: {
-    getUser: rateLimitedResolver(
-      { requestsPerMinute: 20 },
-      async (_, { userId }, { dataSources }) => {
-        const user = await dataSources.usersAPI.getUserById(userId!)
-        return user
-      }
-    ),
-
+    getUser: async (_, { userId }, { dataSources }) => {
+      const user = await dataSources.usersAPI.getUserById(userId!)
+      return user
+    },
```

Comment:
Are we **intentionally** removing the rate limit from this endpoint?

Reply:
No, good catch! I removed it while debugging something